src/jvm/main/org/jetbrains/kotlinx/lincheck/util/JdkVersion.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck.util
+
+
+/**
+ * Represents a set of Java Development Kit (JDK) versions on which the tests are run.
+ */
+internal enum class TestJdkVersion {
```

Comment:
If you move it here from the test infra code, then you need to support all major JDK versions.
Also please rename to `JdkVersion`.